sirepo/auth_db.py

```diff
@@ -252,6 +257,35 @@ def init_model(callback):
         UserDbBase.metadata.create_all(_engine)
 
 
+@contextlib.contextmanager
+def session():
+    init()
+    with sirepo.srcontext.create() as c:
```

Comment:
do we always want to create a context here? it seems like this is a distinct operation.